### ISSUE TITLE
[HOLD for payment 2023-07-17] [$1000] [mWeb-safari] topbar color is different in LHN

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#20516]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. open App
2. check the top bar  of LHN
## Expected Result: 
top bar colour should not be different
## Actual Result: 
top bar colour is different
## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [x] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.26-1
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

<img src="https://github.com/Expensify/App/assets/43996225/8f011b3e-69ef-433f-a42c-f683515b7a75" width="40%" height="40%">

<img src="https://github.com/Expensify/App/assets/43996225/e625e506-d0ed-475b-a3cc-61411acb121b" width="40%" height="40%">

**Expensify/Expensify Issue URL:**
**Issue reported by:** @gadhiyamanan 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1685961858016909

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01b4f1a55f600d32fe</li>
        <li>Upwork Job ID: 1667192033642819584</li>
        <li>Last Price Increase: 2023-06-09</li>
    </ul>
</details>

**Comment by conorpendergrast** [comment]
Nice, thanks!

**Comment by MrJithil** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
mWeb user seeing a different darker topbar color on  iOS safari browser. 

### What is the root cause of that problem?
`theme-color` meta info on head of the html  causing this issue. Currently, the `theme-color` meta being rendered as 

`<meta name="theme-color" content="#061B09">`

### What changes do you think we should make in order to solve the problem?

The `theme-color` meta should be set to `colors.greenHighlightBackground`

Evidence:
![image](https://github.com/Expensify/App/assets/26359740/c262ab55-d19c-40c9-a482-8362af261370)


### What alternative solutions did you explore? (Optional)

**Comment by MrJithil** [comment]
Updated the proposal, because the way we set `meta[name=theme-color]` has been changed as part of the PR - 20436

**Comment by conorpendergrast** [comment]
Proposals are coming in and waiting for review by @thesahindia

**Comment by pecanoro** [comment]
Yeah, I rarely saw websites with this working, I had to try at least 10 until I found one (I think it was snapchat). @Julesssss Do you think this is worth fixing?

**Comment by Julesssss** [comment]
This is something that we fixed on Android, but I think each platform where this occurs might require a native solution. I think the decision to fix this or not depends on the complexity of the solution. I could go either way, but probably not if the solution touches other platforms.

**Comment by pecanoro** [comment]
@thesahindia What do you think about the proposals?

**Comment by pecanoro** [comment]
Assigning @kushu7 as it seems different colors are used in different pages.

**Comment by conorpendergrast** [comment]
Sent contracts to @kushu7 and @thesahindia via Upwork. 

@gadhiyamanan I'm not finding you via Upwork! Can you apply here please? https://www.upwork.com/jobs/~01b4f1a55f600d32fe

Thanks!

**Comment by gadhiyamanan** [comment]
@conorpendergrast applied, Thanks!

**Comment by conorpendergrast** [comment]
Thank you, hired!

**Comment by JmillsExpensify** [comment]
Question: Does this PR also fix the issue on the mobile app? I just ran into this one, same issue but on iOS app instead of mWeb.

**Comment by conorpendergrast** [comment]
Note to self: I'll need to switch the contract in UpWork from @thesahindia to @allroundexperts

**Comment by conorpendergrast** [comment]
Upwork contracts updated!

**Comment by conorpendergrast** [comment]
@allroundexperts I agree

**Comment by conorpendergrast** [comment]
Hah, it really does look better now! 

![IMG_9757](https://github.com/Expensify/App/assets/5802477/f6c94907-7bd4-4212-b70a-8898747f763f)

**Comment by Pujan92** [comment]
@kushu7  Seems regression https://github.com/Expensify/App/issues/21833 from [this PR](https://github.com/Expensify/App/pull/20805) where RGB value we are trying to iterate in wrong method(hexadecimalToRGBArray)

**Comment by conorpendergrast** [comment]
I'm discussing the correct payment process [via internal Slack](https://expensify.slack.com/archives/C01SKUP7QR0/p1688038471574029)

**Comment by conorpendergrast** [comment]
Alright, summary of next steps from a payment perspective, given the regression: 

1. We don’t pay until 7 days after the new PR is deployed to production
2. The bug reporter is paid as normal
3. The C is paid 50% of the job rate, excluding the urgency bonus
4. The C+ is paid 50% of the job rate, excluding the urgency bonus

It looks like we don't have the regression penalties documented anywhere, so I'll do that too

**Comment by situchan** [comment]
It seems first [discovered](https://github.com/Expensify/App/issues/21833) on June 22 and [GH](https://github.com/Expensify/App/issues/21833) created on Jun 28, while [PR](https://github.com/Expensify/App/pull/20805) deployed to production on Jun 22

**Comment by pecanoro** [comment]
@allroundexperts friendly bump for the list above 
@conorpendergrast friendly bump for the payments

🤗

**Comment by conorpendergrast** [comment]
Nice, thanks Roccio. I'll get that paid today

**Comment by conorpendergrast** [comment]
Huh it should be daily now, weird. Moving to daily for the payments step

**Comment by conorpendergrast** [comment]
Payments made, with [these adjustments](https://github.com/Expensify/App/issues/20516#issuecomment-1614808471).

Once the checklist is done, we can close this out

**Comment by conorpendergrast** [comment]
That works for me 👍

**Comment by anmurali** [comment]
Approved $500 @allroundexperts based on https://github.com/Expensify/App/issues/20516#issuecomment-1614808471

**Comment by conorpendergrast** [comment]
@anmurali @allroundexperts I think we double-paid Sibtain for this. He was already paid [via Upwork](https://www.upwork.com/nx/wm/workroom/33963969/overview)

**Comment by conorpendergrast** [comment]
I think the easiest approach is probably that I request a refund via Upwork. Unless I hear otherwise, I'll do that

**Comment by conorpendergrast** [comment]
Nope, my mistake. The Upwork hire was created before we had ND payments, and I didn't double-check before paying. Refund request sent, thank you!

**Comment by conorpendergrast** [comment]
Great, confirmed. Thank you!
